### PR TITLE
SEN-1234 -- about page

### PR DESCRIPTION
**Pull request by noTxt**
**Reviewing:**
- checkout branch
- `bundle exec jekyll serve`
- [visit page](http://localhost:4000/about/)

**Ticket:**
https://cascadeenergy.atlassian.net/browse/SEN-1234

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: page
-title: About
+title: About Us
 ---
 
+A small team of software engineers [devoted](/developer_agreement) to building the [best applications possible](/coding_standards), Cascade Development started in 2013 with the [SENSEI](http://energysensei.com/) product.
 
+Housed within the organization of [Cascade Energy](https://cascadeenergy.com/company/), our development team strives to live the Cascade ideals of *smart, sustained, and measurable* in the context of software.
```
Comment (nackjicholson) [inline about/index.md]: I think you could drop the "in the context of software" and it still reads well.

**Comment by noTxt** [comment]
( @chrispcortez @cberube )

**Comment by noTxt** [comment]
*note:*
If possible, let's not get too hung up on details. It's versioned, in a repo, and easy to deploy &mdash; we always tweak it down the line

**Comment by nackjicholson** [comment]
:ok: :shipit:

**Pull request merged**